evalml/pipelines/components/component_base.py

```diff
@@ -256,7 +257,8 @@ def _handle_nullable_types(self, X=None, y=None):
         Args:
             X (pd.DataFrame, optional): Input data to a component of shape [n_samples, n_features].
                 May contain nullable types.
-            y (pd.Series, optional): The target of length [n_samples]. May contain nullable types.
+            y (pd.Series or pd.DataFrame, optional): The target of length [n_samples] or the unstacked target for a multiseries problem.
+                May contain nullable types.
```

Comment:
Can you be more specific about the multiseries shape? It'll be shape `[n_samples, n_features*n_series]`